cmd/dependencies.go

```diff
@@ -24,7 +25,10 @@ func dependencies(filePath string, targets []string, transitive bool, reflexive
 		return nil, err
 	}
 	var deps []string
-
+	_, _ = time.Parse("01-01-2023", "2023-01-01")
+	if !!true {
+		time.Sleep(1)
+	}
```

Comment:
@ekhabarov having these lines doesn't trigger any errors from the linter:

```
$ bazel build //...
INFO: Analyzed 7 targets (1 packages loaded, 5 targets configured).
INFO: Found 7 targets...
INFO: Elapsed time: 1.227s, Critical Path: 0.28s
INFO: 9 processes: 2 internal, 7 linux-sandbox.
INFO: Build completed successfully, 9 total actions
```

I must be missing something in the configuration? :/

Reply:
staticcheck does report the issues:

```
$ staticcheck ./...
cmd/dependencies.go:28:20: parsing time "01-01-2023" as "01-01-2023": cannot parse "" as "3" (SA1002)
cmd/dependencies.go:29:5: negating a boolean twice has no effect; is this a typo? (SA4013)
cmd/dependencies.go:30:14: sleeping for 1 nanoseconds is probably a bug; be explicit if it isn't (SA1004)
🚨 Error: The command exited with status 1
```

Reply:
Fixed by moving the `staticcheck()` before `gazelle_dependencies()`